skimage/util/points_to_labels.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+
+__all__ = ["label_points"]
+
+
+def label_points(coords, image):
+    """Assign unique integer labels to coordinates on an image mask
+
+    Parameters
+    ----------
+    coords: ndarray
+        An array of 2D coordinates with shape (n, 2)
+    image: ndarray
+        Image for which the mask is created
+
+    Returns
+    -------
+    label_mask: ndarray
+        A 2D mask of zeroes containing unique integer labels at the `coords`
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from skimage.util.points_to_labels import label_points
+    >>> coords = np.array([[0, 1], [2, 2]])
+    >>> image = np.zeros((5, 5), dtype=np.uint32)
+    >>> mask = label_points(coords, image)
+    >>> mask
+    array([[0, 1, 0, 0, 0],
+           [0, 0, 0, 0, 0],
+           [0, 0, 2, 0, 0],
+           [0, 0, 0, 0, 0],
+           [0, 0, 0, 0, 0]], dtype=uint64)
+    """
+    if not isinstance(coords, np.ndarray):
+        raise TypeError("'coords' must be an ndarray")
+
+    if coords.shape[-1] != 2 or coords.ndim != 2:
+        raise ValueError("'coords' must be of shape (n, 2)")
+
+    h, w = image.shape[:2]
+    np_indices = tuple(np.transpose(np.round(coords).astype(np.int)))
```

Comment:
```suggestion
    np_indices = tuple(c for c in np.transpose(np.round(coords)).astype(int))
```
This suggestion avoids conversion of the array to a tuple and instead creates a tuple containing coordinate array views (thus avoiding a copy of the array values). A "tuple of arrays" format like this is what functions like `np.nonzero` return and will work the same for the indexing used below.